tests/component_test.go

```diff
@@ -397,3 +407,77 @@ func Test_07_RuleBindingApplyTest(t *testing.T) {
 //func Test_08_BasicAlertTestExistingProfile(t *testing.T) {
 //
 //}
+
+func TestApplicationProfilePatching_AlwaysFail(t *testing.T) {
```

Comment:
This test needs to be fixed (we expect an error at the end, but we shouldn't get one)